portfolio/src/main/webapp/index.html

```diff
@@ -152,5 +156,8 @@ <h3 class="cat-head">Belyash</h3>
       <p class="email">olgashimanskaia@gmail.com</p>
     </footer>
     <script src="js/script.js"></script>
+    <script async defer
+      src="https://maps.googleapis.com/maps/api/js?key=AIzaSyAMDsO4sw9D4SUQXtP-YA7DzMbcUVdeV80&callback=initMap">
```

Comment:
Because you are making this API key public, please take a look at the [good practices](https://developers.google.com/maps/api-key-best-practices#api_key_table) for API keys. You want to avoid this API to be used massively by someone else, which can go over your usage limit.